README.md

```diff
@@ -1,42 +1,74 @@
-
 Integration Tests for Linux System Roles
 ========================================
 
-A docker container which runs the tests from one of the [linux system
-roles](https://github.com/linux-system-roles) against various operating
-systems.
-
-It starts a virtual machine based on a cloud image of each operating system
-(defined in [images.yml](/images.yml)) and runs playbooks in the roles `test`
-directory against that machine.
+A docker container which runs integration tests for open pull requests on
+[linux system roles](https://linux-system-roles.github.io) repositories. It runs
+all playbooks `test/test_*.yml` of the repository against various virtual
+machines.
 
 To build the container, run
 
-    docker build --tag cockpit/linux-system-roles-test .
+    docker build --tag linux-system-roles-test .
+
+Multiple of these containers can be started in parallel. They try to not step
+on each others feet by synchronizing via GitHub commit statuses.
+
+The test runner needs a `config.json` in the `/config` mount, which specifies
```

Comment:
Is there such a well-known ```config.json```? Worth linking here if so.

Reply:
There is, right above in the `README.md`. Do you think it makes sense to add that as an `example-config.json` in the root of the repository?

I didn't include the one I'm using on OpenShift right now because that includes links to internal RHEL images.